frontend/src/utils/config.js

```diff
@@ -5,8 +5,10 @@
  * Appwrite config object
  */
 export const AppwriteServer = {
-	endpoint : process.env.REACT_APP_ENDPOINT || "http://localhost:80/v1",
+	endpoint : process.env.REACT_APP_ENDPOINT || "http://localhost:81/v1",
```

Comment:
What is the reason for changing this port?

Reply:
I didn't change this. I merged from main. Didn't see that